yapf/yapflib/py3compat.py

```diff
@@ -83,3 +83,7 @@ class ConfigParser(configparser.ConfigParser):
 
     def read_file(self, fp, source=None):
       self.readfp(fp, filename=source)
+
+
+def indent(text, indent):
+    return '\n'.join(indent + l for l in text.split('\n')[:-1])
```

Comment:
Also, why is this function in `py3compat`? That module is for Python 3 compatibility functions only.